ghost/core/core/server/data/migrations/versions/5.31/2023-01-19-07-46-add-mentions-table.js

```diff
@@ -0,0 +1,23 @@
+const {addTable} = require('../../utils');
+
+// stay below 
+const mentionsTable = addTable('mentions', {
```

Comment:
Any reason why we didn't go for `module.exports = addTable(...);`

Reply:
Yeah can we switch to the suggestion above? 🙂